app/components/collectionShow/actions.tsx

```diff
@@ -36,19 +36,21 @@ export function getPapers(params: GetCollectionsPapersParams) {
       const paperResponse = await CollectionAPI.getPapers(params);
 
       dispatch(ActionCreators.addEntity({ entities: paperResponse.entities, result: paperResponse.result }));
-      dispatch(
-        ActionCreators.succeededToGetPapersInCollectionShow({
-          paperIds: paperResponse.result,
-          sort: params.sort,
-          number: paperResponse.number,
-          size: paperResponse.size,
-          first: paperResponse.first,
-          last: paperResponse.last,
-          numberOfElements: paperResponse.numberOfElements,
-          totalPages: paperResponse.totalPages,
-          totalElements: paperResponse.totalElements,
-        })
-      );
+      if (paperResponse.page) {
+        dispatch(
+          ActionCreators.succeededToGetPapersInCollectionShow({
```

Comment:
You can reduce this response. (also should change reducer too.)

You can just send `paperResponse` when reducer is ready